doc/content/concepts/fetchers.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Fetchers"
+description: ""
+weight: 20
+---
+
+## What is it?
+
+Fetchers allow the stack to gain access to external files, such as [Email Templates]({{< ref "/concepts/email-templates.md" >}}), Frequency Plans, Device Data for the Device Repository, Webhook Templates and others.
+
+## Who is it for?
+
+The fetchers are used by users that want to customize certain parts of the stack. They are exposed indirectly through the stack configuration.
+
+### Typical use cases
+
+1. Providing custom resources to the stack, such as custom [Email Templates]({{< ref "/concepts/email-templates.md" >}}) in order to change the behaviour of the stack.
+2. Allowing the stack to run in air-gapped environments, where internet connections are not available constantly and resources must be served locally.
+
+## How does it work?
+
+The fetchers provide the stack with a way to access files which can be hosted remotely (such as on a remote website), locally (by providing a local path to the files) or in memory (by providing the contents of the files directly as part of the configuration).
```

Comment:
I'm not sure we fully support that "as part of the configuration"

Reply:
We don't support memory through config; that's only for testing